### PR TITLE
Test

### PR DESCRIPTION
**Pull request by davidnewhall**
add workflow, remove travis

### CODE REVIEW

**Comment by davidnewhall** [comment]
crap, wrong upstream. sorry. :(

File: go.mod [hunk 1]
```diff
@@ -1,3 +1,3 @@
-module github.com/miolini/datacounter
+module golift.io/datacounter
```
Comment (miolini) [inline go.mod:1]: I cannot use this package name.

File: reader_test.go [hunk 2]
```diff
@@ -1,21 +1,32 @@
...
 import (
 	"bytes"
 	"io"
 	"io/ioutil"
 	"testing"
-)
 
-var data = []byte("Hello, World!")
-var dataLen = uint64(len(data))
+	"golift.io/datacounter"
```
Comment (miolini) [inline reader_test.go:8]: Same, wrong package name.

File: response_writer_test.go [hunk 3]
```diff
@@ -1,23 +1,40 @@
-package datacounter
+package datacounter_test
 
 import (
+	"context"
 	"net/http"
 	"net/http/httptest"
 	"testing"
+
+	"golift.io/datacounter"
```
Comment (miolini) [inline response_writer_test.go:9]: Cannot import this package.

**Review by miolini — CHANGES_REQUESTED** [review]
Hi! I saw your package inside source code. Could you please change it to original.

**Comment by davidnewhall** [comment]
Sorry, I forked your repo and have been maintaining my own copy of this code. I accidentally opened this PR against your repo instead of my own. Probably the second time I've done that. haha sorry.

**Comment by davidnewhall** [comment]
And on that note, you're welcome to copy any of my changes. If you add in the two methods I forked your code to make I'll probably switch back.

**Pull request closed without merging**